plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -785,6 +846,15 @@ public void testListTablesFromSchemaWithBigAmountOfTables()
         assertThat(getQueryRunner().execute("SHOW TABLES FROM mongodb.huge_schema").getRowCount()).isEqualTo(10_000);
     }
 
+    @Test
+    public void testSystemSchemas()
+    {
+        // Ensures that system schemas are inaccessible
+        assertQueryReturnsEmptyResult("SHOW SCHEMAS IN mongodb LIKE 'admin'");
+        assertQueryReturnsEmptyResult("SHOW SCHEMAS IN mongodb LIKE 'config'");
+        assertQueryReturnsEmptyResult("SHOW SCHEMAS IN mongodb LIKE 'local'");
+    }
+
```

Comment:
Please move to the 4th commit "Filter the system schemas in mongodb".

Reply:
Oops seems messed up while working on fixing the table names.